src/index.js

```diff
@@ -51,16 +51,6 @@ function Repo (repoPath, options) {
   this.datastore = stores
                 .datastore
                 .setUp(repoPath, options.stores.datastore, this.locks)
-
-  // TODO: needs https://github.com/ipfs/js-ipfs-repo/issues/6#issuecomment-164650642
-  // this.datastoreLegacy = stores
-  //              .datastore
-  //              .setUp(repoPath, options.stores.datastore, this.locks)
```

Comment:
Let's make a tracking issue for the work then -- that's the proper place for it. Issues embedded in code have a very low visibility and discoverability.